ci/terraform/oidc/variables.tf

```
@@ -535,6 +535,18 @@ variable "orch_authorisation_enabled" {
   default     = false
 }
 
+variable "orch_backchannel_logout_enabled" {
+  description = "Flag to disable routing back channel logout traffic to the authentication account"
+  type        = bool
+  default     = false
+}
+
+variable "orch_spot_response_enabled" {
+  description = "Flag to disable routing spot response traffic to the authentication account"
+  type        = bool
+  default     = false
+}
+
```

I find having them be `flag_enabled` with a description that says "disable" a bit confusing - maybe switch it round? Think it is more consistent with the other flags too
```suggestion
variable "orch_backchannel_logout_enabled" {
  description = "Flag to enable routing back channel logout traffic to the authentication account"
  type        = bool
  default     = false
}

variable "orch_spot_response_enabled" {
  description = "Flag to enable routing spot response traffic to the authentication account"
  type        = bool
  default     = false
}

```

---

Unless these are disable flags in which case maybe change the names to `flag_disabled`